packages/accordions/src/styled/accordion/StyledRotateIcon.ts

```diff
@@ -20,7 +20,7 @@ const colorStyles = (props: ThemeProps<DefaultTheme> & any) => {
   const showColor = props.isCollapsible || !props.isRotated;
 
   return css`
-    color: ${showColor ? props.isHovered && color : getColor('neutralHue', 400, props.theme)};
+    color: ${showColor ? props.isHovered && color : getColor('neutralHue', 600, props.theme)};
```

Comment:
These `colorStyles` don't appear to be getting applied.

Reply:
This boolean/ternary logic was invalid. We now show `grey-600` as the default.